test/connectivity/modules/NetworkController/INetworkController.cs

```diff
@@ -4,7 +4,7 @@ namespace NetworkController
     using System.Threading;
     using System.Threading.Tasks;
 
-    interface IController
+    interface INetworkController
```

Comment:
please rename SetStatus and GetStatus to SetStatusAsync and GetStatusAsync respectively.